src/storage/src/hummock/sstable_store.rs

```diff
@@ -492,37 +492,41 @@ impl SstableStore {
         self.meta_cache.get_memory_usage() as u64
     }
 
-    pub async fn get_stream(
+    pub async fn get_stream_by_position(
```

Comment:
The caller of `get_stream_by_position` must passed in a Some(block_idx), we can remove this Option